dev/io.openliberty.jakarta.interceptor.2.1/bnd.bnd

```diff
@@ -0,0 +1,25 @@
+#*******************************************************************************
+# Copyright (c) 2020, 2022 IBM Corporation and others.
```

Comment:
Shouldn't it just be 2022?

Reply:
Well grey area ... the file was originally copied from the 2.0 version and then modified for 2.2 ... so technically I think the dates are correct